samples/routing/fowarding-address/sample.md

```diff
@@ -2,55 +2,55 @@
 title: Forwarding address
 summary: Implementing forwarding address for in-flight messages during handler migration
 component: Core
-reviewed: 2017-09-27
+reviewed: 2019-06-25
 tags:
 - Routing
 ---
 
-When a handler is migrated to a new endpoint there can be outstanding messages in-flight. When these messages arrive at the original endpoint they need to be re-routed to the new endpoint.
+When a handler is moved to a new endpoint, there may still be "in-flight" messages, bound for the original endpoint. When those messages arrive at the original endpoint, they need to be re-routed to the new endpoint.
 
-NServiceBus can already be configured to [forward a copy of every message that it processes to another address](/nservicebus/messaging/forwarding.md). This sample is different in a few key ways:
+An endpoint may be configured to [forward a copy of every successfully processed message to another endpoint](/nservicebus/messaging/forwarding.md). This sample is different in a few key ways:
 
  1. This sample only forwards messages which have a configured forwarding address. Each message type can have a different forwarding address. The built-in forwarding forwards every message to a single destination address.
  1. This sample forwards messages to a logical endpoint, rather than a physical address. This allows it to interact with [sender-side distribution](/transports/msmq/sender-side-distribution.md) if required.
 
 
-## Running the project
+## Running the sample
 
- 1. Start all the projects by hitting F5.
- 1. The text `Endpoint Started. Press s to send a very important message. Any other key to exit` should be displayed in the Sender's console window.
- 1. Press S to send a message.
- 1. The message will be processed by OriginalDestination
- 1. The message will also be processed by NewDestination
+1. Open the solution in Visual Studio.
+1. Press F5.
+1. Follow the instructions in sender's console window.
```

Comment:
```suggestion
1. Follow the instructions in sender's console window to send a message to the OriginalDestination endpoint.
```

Otherwise "the message" which is the context for the next two points comes out of nowhere.